manuscript.md

```diff
@@ -43,6 +43,8 @@ landscape modification [@Edwards2021TroLan; @Magioli2021DefLea;
 possible in the prediction of networks, both extant (imputation) and future
 (forecasting).
 
+**Paragraph on disease goes here**
```

Comment:
Understanding disease transmission requires different methodological approaches that deal with interaction complexity. Networks assessment of vector-borne diseases, which have several hosts and intricate transmission cycles, is one of the examples in the field of disease ecology research. Few studies have implemented machine learning techniques for forecasting the incident cases of dengue, Chagas disease, rickettsiosis, leishmaniasis, as well as livestock and wildlife infectious diseases. 
  (Trop. Med. Infect. Dis. 2020, 5, 64; doi:10.3390/tropicalmed5020064 ; Phil. Trans. R. Soc. B 370: 20140107. http://dx.doi.org/10.1098/rstb.2014.0107  ;  PLOS Neglected Tropical Diseases | https://doi.org/10.1371/journal.pntd.0008056 September 24, 2020 ; Parasitology, Page 1 of 13. doi:10.1017/S0031182016002468  ; PLoS ONE 4(5): e5725. doi:10.1371/journal.pone.0005725 )